src/main/java/org/openrewrite/java/testing/mockito/MockitoWhenOnStaticToMockStatic.java

```diff
@@ -37,13 +37,15 @@
 
 public class MockitoWhenOnStaticToMockStatic extends Recipe {
 
-    private static final AnnotationMatcher JUNIT_ANNOTATION = new AnnotationMatcher("org.junit.*");
+    private static final AnnotationMatcher JUNIT_4_ANNOTATION = new AnnotationMatcher("org.junit.*");
+    private static final AnnotationMatcher JUNIT_5_ANNOTATION = new AnnotationMatcher("org.junit.jupiter.api.*");
     private static final AnnotationMatcher TESTNG_ANNOTATION = new AnnotationMatcher("org.testng.annotations.*");
 
     private static final AnnotationMatcher BEFORE = new AnnotationMatcher("org..Before*");
     private static final AnnotationMatcher BEFORE_CLASS = new AnnotationMatcher("org..BeforeClass");
+    private static final AnnotationMatcher BEFORE_ALL = new AnnotationMatcher("org..BeforeAll");
```

Comment:
Would `BeforeAll` not already be picked up by the wildcard in `"org..Before*"` ?